resolwe_bio/listener_plugins/apps.py

```diff
@@ -0,0 +1,25 @@
+""".. Ignore pydocstyle D400.
+
+========================
+Resolwe listener plugins
+========================
+
+"""
+from django.apps import AppConfig
+
+
+class ListenerPluginConfig(AppConfig):
```

Comment:
Is there a reason why you don't add this configuration into the existing `kb` app?

Reply:
I did now.